Wiki/starting/starting-leaving.md

```diff
@@ -0,0 +1,146 @@
+- please delete anything that is not required
+- please send an email to the CMS team (copied on this email), detailing the nature of what will remain. This should outline major directories and explain:
+	- what they contain 
+	- why they’re important to keep
+	- what is required to access them - which repository / scripts?
+	- if this data is stored anywhere else
+	- when can they be deleted (date)
+	- where this data should be stored when your account is shut down (please consult with a CLEX CI)
+	- who in CLEX should have access to this data?
+	- please also let the CMS team if compute access will be required after your contract ends
+ - please reply to both the admin and CMS teams with future contact details when you have them
+
+The CMS team can help with tools and techniques to help you manage your data. Please understand that storage is extremely limited, so that a lack of response to the questions above will likely result in your data being archived as soon as your contract ends.
+
+Regards,
+
+The CLEX Admin Team
+```
+
+While you will receive the email above to remind you what needs to be done, sorting through your files might take longer than you think and it is important to be organised at any stage of your project. So please consider running the suggested actions at different stages of your project whne appropriate.
+
+You might need to discuss the following questions with your supervisor:
+
+* Will you require access to the files after you leave (e.g. for a paper review)?
+* What files will be useful to others? Should they be published?
+* What files won't be useful to others? Should they be archived or deleted?
+
+```{note}
+If you need specific advice on how to actually transfer or archive your files, you can always ask for assistance to our helpdesk: cws_help@nci.org.au.
+```
+
+## What to do
+
+While what exactly should be kept, removed or published will depend on your specific project and working group. There are some principles that can be applied to any situation.
+ 
+### To keep, always
+
+**Codes**
+
+No matter what the details of your projects are you should always keep the code you used. This means keeping a reference to the exact version and document how you used codes developed by others (e.g. climate models). If you modified them in any way this should also be documented. 
+
+If you modfication are extensive or you have written the code yourself then you should make it available via GitHub and publish it (Zenodo is the most indicated for this).
+
+```{note}
+It is absolutely fine to create a repository per project or paper with all your codes in. You can then use the README file from the GitHub repository to explain how to reproduce your results. Don't forget to clearly reference everything one might need in addition to this repository. Or you can have a repository per code especially if you envision you'll reuse the same code for other work.
+```
+
+**Configuration files for running the codes and some input files**
```

Comment:
```suggestion
**Configuration files and certain input files**
```

Reply:
Maybe we should rename this completely to "Files needed to run the code" or something similar

Reply:
done